lib/idStorage.js

```diff
@@ -0,0 +1,61 @@
+const storage = new Map();
```

Comment:
what does this turn into after running through babel?  A little worried that this bloats the code